### PR TITLE
Customized volume panels[2/3]

### PR DESCRIPTION
**Pull request by Thatisjigen**

### CODE REVIEW

File: res/values/cr_strings.xml [hunk 1]
```diff
@@ -2687,4 +2687,19 @@
...
+    <string name="volume_panel">Volume Panel Style</string>
+    <string name="volume_panel_stock">Stock</string>
+    <string name="volume_panel_aosp">AOSP</string>
+    <string name="volume_panel_compact">Compact</string>
+    <string name="volume_panel_oreo">Oreo</string>
+    <string name="volume_panel_tiled">Tiled</string>
+
+    <!--  Volume Panel Plugin Values   -->
+    <string name="volume_panel_stock_val">com.android.systemui.volume</string>
+    <string name="volume_panel_aosp_val">co.potatoproject.plugin.volume.aosp</string>
```
Comment (gwolf2u) [inline res/values/cr_strings.xml:2701]: maybe rename those?

**Comment by gwolf2u** [comment]
also resolve conflicts

File: res/values/cr_strings.xml (hunk 1 above)
Comment (Thatisjigen) [reply res/values/cr_strings.xml:2701]: POSP team asked to keep package names as is

**Comment by gwolf2u** [comment]
DNM for now

**Pull request closed without merging**